### PR TITLE
added tangent aero with @jorgepz

### PR DESCRIPTION
**Pull request by mvanzulli**
This PR close a subtask of #363. A validation example will be added further on. Note that I've added a new field in `elemTypeAero` field 

```octave
computeAeroTangMatrix = false ;
elements(2).elemTypeAero   = [0 dext 0 numGaussPoints computeAeroTangMatrix];
```

According to my non-extensive tests the computation of the tangent matrix can induce a significant performance penalty. But now the nonLinearAerodynamics static example resits until 100m/s :100: 

Thanks to @jorgepz and @TheChikenfly  for the discussion.

### CODE REVIEW

File: examples/linearAerodynamics/linearAerodynamics.m [hunk 1]
```diff
@@ -71,9 +71,10 @@
...
+%md Next the _elemTypeAero_ field contain the information of the chord vector. This vector is defined first considering the orientation of the cross section set up in lift, drag and moment experiments, and then how that cross section is located for the example. In this case the orientation of the chord vector is along $y$. In general note that the chord vector $t_{ch}$ must be given in reference (non canonical configurations). In this example the cable is oriented along $y$ so the direction will be $[0~1~0]$ as it is shown in Fig 1. Also the length of the chord is added to the norm of the chord vector, for cylindrical cantilever beams is $d_{ext}$. The aerodynamic tangent matrix is the last boolean element of the vector `elemTypeAero`, and in this case is set as false.  All this information is added into _elemTypeAero_ field of `elements` struct such that:
 numGaussPoints  = 4 ; 
-elements(2).elemTypeAero   = [0 dext 0 numGaussPoints];
+computeAeroTangMatrix = false ;
+elements(2).elemTypeAero   = [0 dext 0 numGaussPoints computeAeroTangMatrix];
```
[the remark this answers is not in the document]
Comment (jorgepz) [reply examples/linearAerodynamics/linearAerodynamics.m:77]: booleans are automatically converted to floats when inserted in a floats vector in octave. i think this is ok by the moment.

**Review by jorgepz — APPROVED** [review]

File: examples/linearAerodynamics/linearAerodynamics.m (hunk 1 above)
[the remark this answers is not in the document]
Comment (mvanzulli) [reply examples/linearAerodynamics/linearAerodynamics.m:77]: Okay :+1:

**Comment by TheChikenfly** [comment]
nice work!

**Comment by jorgepz** [comment]
Thanks!

**Pull request merged**